rskj-core/src/main/java/co/rsk/peg/storage/FederationStorageIndexKey.java

```diff
@@ -25,7 +25,8 @@ public enum FederationStorageIndexKey {
     NEW_FEDERATION_FORMAT_VERSION("newFederationFormatVersion"),
     OLD_FEDERATION_FORMAT_VERSION("oldFederationFormatVersion"),
     PENDING_FEDERATION_FORMAT_VERSION("pendingFederationFormatVersion"),
-    PROPOSED_FEDERATION_FORMAT_VERSION("proposedFederationFormatVersion")
+    PROPOSED_FEDERATION_FORMAT_VERSION("proposedFederationFormatVersion"),
+    FUND_TX_HASH_UNSIGNED("fundTxHashUnsigned")
```

Comment:
@julia-zack remind me, but I think we talked about putting an SVP prefix to these values right? I think that was even part of the RSKIP.

fundTransaction by itself doesn't say much

Reply:
Updated.